lambda_function/package/populate.py

```diff
@@ -0,0 +1,88 @@
+import os
```

Comment:
Please create a folder called 'post-deployment-scripts' (just like scripts folder) and move this python there.

Reply:
resolved